src/extensions/default/bramble/stylesheets/lightTheme.less

```diff
@@ -21,7 +21,7 @@ span.cm-tag			{ color: #002cb8; }
 span.cm-attribute	{ color: #008d5c; }
 span.cm-property	{ color: #008d5c; } /* property after string */
 
-span.cm-comment		{ color: #779cac; }
+span.cm-comment		{ color: #B1B1B1; }
 span.cm-error		{ color: #ff3da5; }
 span.cm-keyword		{ color: #002ea3; }
 span.cm-variable	{ color: #0056ba; }
```

Comment:
Odd indents going on in the previous block, can you normalize for 4-space indent?